napari-{{cookiecutter.plugin_name}}/setup.py

```diff
@@ -44,14 +44,13 @@ def read(fname):
     classifiers=[
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
+        "Framework :: napari"
```

Comment:
Erm, there is a comma missing... ;) Not to mention double quotes 🤮 😜

Reply:
thanks will update.  but aren't the double quotes the standard?

Reply:
https://pypi.org/classifiers/

Reply:
☝️ ignore this :) I'm an idiot (as discussed on zulip).  my brain went to "double colons" for some reason :)